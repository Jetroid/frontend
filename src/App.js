import React from "react"; // eslint-disable-line

import { IntlProvider, addLocaleData } from "react-intl";
import en from "react-intl/locale-data/en";
import fr from "react-intl/locale-data/fr";
import es from "react-intl/locale-data/es";
import Layout from "./Layout";
import authService from "./utils/AuthService";
import { getFirstBrowserLanguage, getBestMatchingMessages } from "./utils/l10n";
import createBrowserHistory from "history/createBrowserHistory";
import { BrowserRouter } from "react-router-dom";

const history = createBrowserHistory();

addLocaleData([...en, ...fr, ...es]);

export default class App extends React.Component {
  render() {
    let locale = getFirstBrowserLanguage();
    let messages = getBestMatchingMessages(locale);
    return (
      <IntlProvider locale={locale} messages={messages}>
        <BrowserRouter history={history}>
          <Layout history={history} auth={authService} />
        </BrowserRouter>
      </IntlProvider>
    );
  }
}
