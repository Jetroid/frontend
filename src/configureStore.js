import {compose, createStore, applyMiddleware} from 'redux'
import * as storage from 'redux-storage'
import createEngine from 'redux-storage-engine-localstorage'
const engine = createEngine('my-save-key')
// import persistState from "redux-localstorage"
// import promiseMiddleware from 'redux-promise-middleware'
import thunkMiddleware from 'redux-thunk'
import createLogger from 'redux-logger'
import rootReducer from './reducers'

module.exports = function configureStore () {
  const storageMiddleware = storage.createMiddleware(engine)
  // const loggerMiddleware = createLogger()

  const initialState = {}
  // const withStorage = applyMiddleware(storageMiddleware)(createStore);
  // const withPromise = applyMiddleware(promiseMiddleware)(withStorage);
  let store = createStore(
    rootReducer,
    initialState,
    compose(
      applyMiddleware(
        thunkMiddleware,
        // loggerMiddleware,
        storageMiddleware
      ),
      window.devToolsExtension ? window.devToolsExtension() : f => f
    )
  )

  // if (process.env.NODE_ENV !== 'production') {
  if (module.hot) {
    // Enable Webpack hot module replacement for reducers
    module.hot.accept('./reducers', () => {
      var nextRootReducer = require('./reducers')
      store.replaceReducer(nextRootReducer)
    })
  }
  // }

  return store
}
