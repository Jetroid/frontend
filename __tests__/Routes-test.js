import { render, unmountComponentAtNode } from "react-dom";
import React from "react";
import { Route, MemoryRouter } from "react-router-dom";
import { Simulate } from "react-addons-test-utils";
import { Provider } from "react-redux";
import searchData from "./search_data.json";
import countryData from "./country_data.json";
import afterPromises from "../src/helpers/afterPromises";
import App from "../src/App";

let fetchMock = require("fetch-mock");
import configureStore from "../src/configureStore";
import injectTapEventPlugin from "react-tap-event-plugin";
injectTapEventPlugin();

let store = configureStore();
fetchMock.get(
  process.env.REACT_APP_API_URL +
    "/search?query=&selectedCategory=All&sortingMethod=chronological",
  JSON.stringify({ data: searchData })
);
fetchMock.get(
  process.env.REACT_APP_API_URL + "/case/countsByCountry",
  JSON.stringify(countryData)
);

// a way to render any part of your app inside a MemoryRouter
// you pass it a list of steps to execute when the location
// changes, it will call back to you with stuff like
// `match` and `location`, and `history` so you can control
// the flow and make assertions.
const renderTestSequence = (
  {
    initialEntries,
    initialIndex,
    subject: Subject,
    steps
  }
) => {
  const div = document.createElement("div");
  class Assert extends React.Component {
    componentDidMount() {
      this.assert();
    }
    componentDidUpdate() {
      this.assert();
    }
    assert() {
      const nextStep = steps.shift();
      if (nextStep) {
        nextStep({ ...this.props, div });
      } else {
        unmountComponentAtNode(div);
      }
    }
    render() {
      return this.props.children;
    }
  }
  class Test extends React.Component {
    render() {
      return (
        <MemoryRouter
          initialIndex={initialIndex}
          initialEntries={initialEntries}
        >
          <Route
            render={props => (
              <Assert {...props}>
                <Subject />
              </Assert>
            )}
          />
        </MemoryRouter>
      );
    }
  }
  render(<Test />, div);
};

const MyProvider = () => (
  <Provider store={store}>
    <App />
  </Provider>
);
// the actual test!
it("navigates around", done => {
  try {
    afterPromises(done, () => {
      renderTestSequence({
        // tell it the subject you're testing
        subject: MyProvider,
        // and the steps to execute each time the location changes
        steps: [
          // initial render
          ({ history, div }) => {
            // assert the screen says what we think it should
            expect(div.innerHTML).toContain("Participedia");
            // now we can imperatively navigate as the test
            //   Simulate.click(div.querySelector(".menu-icon"), {
            //     button: 0
            //   });
            //   // history.push("/dashboard");
            // },
            // // second render from same location (after click)
            // ({ div }) => {
            //   expect(div.innerHTML).toContain("Add New");
            //   //   console.log(div.innerHTML);
            //   //   console.assert(div.innerHTML.match(/Dashboard/));
            //   //   // or we can simulate clicks on Links instead of
            //   //   // using history.push
            //   Simulate.click(div.querySelector(".menu-icon"), {
            //     button: 0
            //   });
            // },
            // // final render
            // ({ location }) => {
            //   expect(div.innerHTML).toNotContain("Add New");

            //   //   console.assert(location.pathname === "/");
            //   //   // you'll want something like `done()` so your test
            //   //   // fails if you never make it here.
            //   done();
          }
        ]
      });
    });
  } finally {
    console.log(fetchMock.calls("*"));
  }
});
