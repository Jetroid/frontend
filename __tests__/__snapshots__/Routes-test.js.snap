exports[`test App 1`] = `
<div>
  <div
    className="nav-bar-component">
    <div
      className="nav-bar-wrapper">
      <div
        className="logo-area">
        <a
          className="menu-icon"
          onClick={[Function]}>
          <img
            alt=""
            src="menu-icon.png" />
        </a>
        <a
          className="logo"
          href="/"
          onClick={[Function]}>
          <img
            alt="Go Home"
            src="pp-logo.png" />
        </a>
      </div>
      <div
        className="search-box-area">
        <input
          className="search-bar"
          onChange={[Function]}
          onKeyUp={[Function]}
          placeholder="Search"
          type="text"
          value="" />
      </div>
      <a
        href="/quick-submit"
        onClick={[Function]}>
        <div
          className="createButton" />
      </a>
      <div
        className="loginButton">
        <button
          disabled={false}
          href={undefined}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchStart={[Function]}
          onTouchTap={[Function]}
          style={
            Object {
              "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
              "backgroundColor": "rgba(0, 0, 0, 0)",
              "border": 10,
              "borderRadius": 2,
              "boxSizing": "border-box",
              "color": "rgba(0, 0, 0, 0.87)",
              "cursor": "pointer",
              "display": "inline-block",
              "fontFamily": "Roboto, sans-serif",
              "fontSize": "inherit",
              "fontWeight": "inherit",
              "height": 36,
              "lineHeight": "36px",
              "margin": 0,
              "minWidth": 88,
              "muiPrepared": true,
              "outline": "none",
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "textAlign": "center",
              "textDecoration": "none",
              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "userSelect": "none",
              "verticalAlign": null,
              "zIndex": 1,
            }
          }
          tabIndex={0}
          type="button">
          <div
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}>
            <span
              style={
                Object {
                  "fontSize": 14,
                  "fontWeight": 500,
                  "letterSpacing": 0,
                  "muiPrepared": true,
                  "paddingLeft": 16,
                  "paddingRight": 16,
                  "position": "relative",
                  "textTransform": "uppercase",
                  "verticalAlign": "middle",
                }
              }>
              Login
            </span>
          </div>
        </button>
      </div>
    </div>
  </div>
  <div
    className="drawer"
    style={undefined}>
    <div
      className={undefined}
      onTouchTap={[Function]}
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
          "backgroundColor": "rgba(0, 0, 0, 0.54)",
          "height": "100%",
          "left": "-100%",
          "muiPrepared": true,
          "opacity": 0,
          "pointerEvents": "none",
          "position": "fixed",
          "top": 0,
          "transform": "translateZ(0)",
          "transition": "left 0ms cubic-bezier(0.23, 1, 0.32, 1) 400ms, opacity 400ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "width": "100%",
          "willChange": "opacity",
          "zIndex": 1200,
        }
      } />
    <div
      className={undefined}
      style={
        Object {
          "WebkitOverflowScrolling": "touch",
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "#ffffff",
          "borderRadius": "0px",
          "boxShadow": "0 3px 10px rgba(0, 0, 0, 0.16),
                 0 3px 10px rgba(0, 0, 0, 0.23)",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "fontFamily": "Roboto, sans-serif",
          "height": "100%",
          "left": 0,
          "muiPrepared": true,
          "overflow": "auto",
          "position": "fixed",
          "top": 0,
          "transform": "translate(-310px, 0)",
          "transition": "transform 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "width": 300,
          "zIndex": 1300,
        }
      } />
  </div>
  <div
    className="contentArea">
    <div
      className="home">
      <Map />
      <div
        className="main-contents">
        <div
          className="search-results-component container-fluid">
          <div
            className="sidepanel hidden-sm-down col col-md-3">
            <div
              className="sorting-options">
              <p
                className="current-sorting-selection"
                onClick={[Function]}>
                Most Recent
              </p>
              <div
                className="unselected"
                onClick={[Function]}>
                Featured
              </div>
              <div
                className="selected"
                onClick={[Function]}>
                Most Recent
              </div>
              <div
                className="unselected"
                onClick={[Function]}>
                Alphabetical
              </div>
            </div>
            <div
              className="featured-searches-area">
              <div
                className="featured-searches-header">
                Featured Searches
              </div>
              <div
                className="featured-searches">
                <div>
                  <div>
                    <a
                      href="/search?query=participatory%2520budgeting"
                      onClick={[Function]}>
                      Mention \"Participatory Budgeting\"
                    </a>
                  </div>
                  <div>
                    <a
                      href="/search?tag=infrastructure"
                      onClick={[Function]}>
                      Tagged as: Infrastructure
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            className="col col-md-9">
            <div
              className="clearfix search-actions-area">
              <div
                className="filters hidden-xs-down">
                <div
                  className="selected"
                  onClick={[Function]}>
                  All
                </div>
                <div
                  className="unselected"
                  onClick={[Function]}>
                  News
                </div>
                <div
                  className="unselected"
                  onClick={[Function]}>
                  cases
                </div>
                <div
                  className="unselected"
                  href="#"
                  onClick={[Function]}>
                  methods
                </div>
                <div
                  className="unselected"
                  href="#"
                  onClick={[Function]}>
                  organizations
                </div>
              </div>
              <select
                className="mobile-select hidden-sm-up"
                onChange={[Function]}
                value="All">
                <option
                  value="All">
                  All
                </option>
                <option
                  value="News">
                  News
                </option>
                <option
                  value="Cases">
                  cases
                </option>
                <option
                  value="Methods">
                  methods
                </option>
                <option
                  value="Organizations">
                  organizations
                </option>
              </select>
              <div
                className="view-types hidden-sm-down">
                <div
                  className="selected"
                  onClick={[Function]}>
                  <img
                    alt=""
                    className="grid-icon"
                    src="pp-search-grid-icon.png" />
                  <img
                    alt=""
                    className="grid-icon"
                    src="pp-search-grid-icon-active.png" />
                </div>
                <div
                  className="unselected"
                  onClick={[Function]}>
                  <img
                    alt=""
                    className="list-icon"
                    src="pp-search-list-icon.png" />
                  <img
                    alt=""
                    className="list-icon"
                    src="pp-search-list-icon-active.png" />
                </div>
                <div
                  onClick={[Function]}>
                  <img
                    alt=""
                    className="dl-icon"
                    src="../img/pp-search-dl-icon.png" />
                </div>
              </div>
            </div>
            <div>
              <h3>
                Searching for
                Â 
                
                <div
                  className="spinner">
                  <ChasingDots
                    size={50} />
                </div>
              </h3>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="footer-component">
    <div
      className="expanded-footer container-fluid">
      <div
        className="row">
        <div
          className="col col-lg-6">
          <div
            className="column list">
            <a
              href="/about"
              onClick={[Function]}>
              About
            </a>
            <a
              href="/research"
              onClick={[Function]}>
              Research
            </a>
            <a
              href="/teaching"
              onClick={[Function]}>
              Teaching
            </a>
            <a
              href="/experiments"
              onClick={[Function]}>
              Experiments
            </a>
          </div>
          <div
            className="column list">
            <a
              href="#">
              cases
            </a>
            <a
              href="#">
              methods
            </a>
            <a
              href="#">
              organizations
            </a>
            <a
              href="#">
              Users
            </a>
          </div>
          <div
            className="column list">
            <a
              href="#">
              News
            </a>
            <a
              href="?help"
              onClick={[Function]}>
              Help
            </a>
            <a
              href="#">
              Contact
            </a>
          </div>
        </div>
        <div
          className="col col-lg-6">
          <div
            className="column xl">
            <h4>
              Participedia
            </h4>
            <p>
              A global community sharing knowledge and stories about public participation
            </p>
          </div>
          <div
            className="column right">
            <div
              className="SocialMediaShareButton SocialMediaShareButton--facebook social-icons"
              onClick={[Function]}
              style={Object {}}>
              <img
                alt=""
                src="pp-social-fb.png" />
            </div>
            <div
              className="SocialMediaShareButton SocialMediaShareButton--twitter social-icons"
              onClick={[Function]}
              style={Object {}}>
              <img
                alt=""
                src="pp-social-tw.png" />
            </div>
            <div
              className="copyright-area">
              <img
                alt=""
                src="cc-icon.png" />
              <span>
                Participedia 2017
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`test Searching for a word 1`] = `
<div>
  <div
    className="nav-bar-component">
    <div
      className="nav-bar-wrapper">
      <div
        className="logo-area">
        <a
          className="menu-icon"
          onClick={[Function]}>
          <img
            alt=""
            src="menu-icon.png" />
        </a>
        <a
          className="logo"
          href="/"
          onClick={[Function]}>
          <img
            alt="Go Home"
            src="pp-logo.png" />
        </a>
      </div>
      <div
        className="search-box-area">
        <input
          className="search-bar"
          onChange={[Function]}
          onKeyUp={[Function]}
          placeholder="Search"
          type="text"
          value="" />
      </div>
      <a
        href="/quick-submit"
        onClick={[Function]}>
        <div
          className="createButton" />
      </a>
      <div
        className="loginButton">
        <button
          disabled={false}
          href={undefined}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchStart={[Function]}
          onTouchTap={[Function]}
          style={
            Object {
              "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
              "backgroundColor": "rgba(0, 0, 0, 0)",
              "border": 10,
              "borderRadius": 2,
              "boxSizing": "border-box",
              "color": "rgba(0, 0, 0, 0.87)",
              "cursor": "pointer",
              "display": "inline-block",
              "fontFamily": "Roboto, sans-serif",
              "fontSize": "inherit",
              "fontWeight": "inherit",
              "height": 36,
              "lineHeight": "36px",
              "margin": 0,
              "minWidth": 88,
              "muiPrepared": true,
              "outline": "none",
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "textAlign": "center",
              "textDecoration": "none",
              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "userSelect": "none",
              "verticalAlign": null,
              "zIndex": 1,
            }
          }
          tabIndex={0}
          type="button">
          <div
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}>
            <span
              style={
                Object {
                  "fontSize": 14,
                  "fontWeight": 500,
                  "letterSpacing": 0,
                  "muiPrepared": true,
                  "paddingLeft": 16,
                  "paddingRight": 16,
                  "position": "relative",
                  "textTransform": "uppercase",
                  "verticalAlign": "middle",
                }
              }>
              Login
            </span>
          </div>
        </button>
      </div>
    </div>
  </div>
  <div
    className="drawer"
    style={undefined}>
    <div
      className={undefined}
      onTouchTap={[Function]}
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
          "backgroundColor": "rgba(0, 0, 0, 0.54)",
          "height": "100%",
          "left": "-100%",
          "muiPrepared": true,
          "opacity": 0,
          "pointerEvents": "none",
          "position": "fixed",
          "top": 0,
          "transform": "translateZ(0)",
          "transition": "left 0ms cubic-bezier(0.23, 1, 0.32, 1) 400ms, opacity 400ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "width": "100%",
          "willChange": "opacity",
          "zIndex": 1200,
        }
      } />
    <div
      className={undefined}
      style={
        Object {
          "WebkitOverflowScrolling": "touch",
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "#ffffff",
          "borderRadius": "0px",
          "boxShadow": "0 3px 10px rgba(0, 0, 0, 0.16),
                 0 3px 10px rgba(0, 0, 0, 0.23)",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "fontFamily": "Roboto, sans-serif",
          "height": "100%",
          "left": 0,
          "muiPrepared": true,
          "overflow": "auto",
          "position": "fixed",
          "top": 0,
          "transform": "translate(-310px, 0)",
          "transition": "transform 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "width": 300,
          "zIndex": 1300,
        }
      } />
  </div>
  <div
    className="contentArea">
    <div
      className="home">
      <Map />
      <div
        className="main-contents">
        <div
          className="search-results-component container-fluid">
          <div
            className="sidepanel hidden-sm-down col col-md-3">
            <div
              className="sorting-options">
              <p
                className="current-sorting-selection"
                onClick={[Function]}>
                Most Recent
              </p>
              <div
                className="unselected"
                onClick={[Function]}>
                Featured
              </div>
              <div
                className="selected"
                onClick={[Function]}>
                Most Recent
              </div>
              <div
                className="unselected"
                onClick={[Function]}>
                Alphabetical
              </div>
            </div>
            <div
              className="featured-searches-area">
              <div
                className="featured-searches-header">
                Featured Searches
              </div>
              <div
                className="featured-searches">
                <div>
                  <div>
                    <a
                      href="/search?query=participatory%2520budgeting"
                      onClick={[Function]}>
                      Mention \"Participatory Budgeting\"
                    </a>
                  </div>
                  <div>
                    <a
                      href="/search?tag=infrastructure"
                      onClick={[Function]}>
                      Tagged as: Infrastructure
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            className="col col-md-9">
            <div
              className="clearfix search-actions-area">
              <div
                className="filters hidden-xs-down">
                <div
                  className="selected"
                  onClick={[Function]}>
                  All
                </div>
                <div
                  className="unselected"
                  onClick={[Function]}>
                  News
                </div>
                <div
                  className="unselected"
                  onClick={[Function]}>
                  cases
                </div>
                <div
                  className="unselected"
                  href="#"
                  onClick={[Function]}>
                  methods
                </div>
                <div
                  className="unselected"
                  href="#"
                  onClick={[Function]}>
                  organizations
                </div>
              </div>
              <select
                className="mobile-select hidden-sm-up"
                onChange={[Function]}
                value="All">
                <option
                  value="All">
                  All
                </option>
                <option
                  value="News">
                  News
                </option>
                <option
                  value="Cases">
                  cases
                </option>
                <option
                  value="Methods">
                  methods
                </option>
                <option
                  value="Organizations">
                  organizations
                </option>
              </select>
              <div
                className="view-types hidden-sm-down">
                <div
                  className="selected"
                  onClick={[Function]}>
                  <img
                    alt=""
                    className="grid-icon"
                    src="pp-search-grid-icon.png" />
                  <img
                    alt=""
                    className="grid-icon"
                    src="pp-search-grid-icon-active.png" />
                </div>
                <div
                  className="unselected"
                  onClick={[Function]}>
                  <img
                    alt=""
                    className="list-icon"
                    src="pp-search-list-icon.png" />
                  <img
                    alt=""
                    className="list-icon"
                    src="pp-search-list-icon-active.png" />
                </div>
                <div
                  onClick={[Function]}>
                  <img
                    alt=""
                    className="dl-icon"
                    src="../img/pp-search-dl-icon.png" />
                </div>
              </div>
            </div>
            <div>
              <h3>
                Searching for
                Â 
                
                <div
                  className="spinner">
                  <ChasingDots
                    size={50} />
                </div>
              </h3>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="footer-component">
    <div
      className="expanded-footer container-fluid">
      <div
        className="row">
        <div
          className="col col-lg-6">
          <div
            className="column list">
            <a
              href="/about"
              onClick={[Function]}>
              About
            </a>
            <a
              href="/research"
              onClick={[Function]}>
              Research
            </a>
            <a
              href="/teaching"
              onClick={[Function]}>
              Teaching
            </a>
            <a
              href="/experiments"
              onClick={[Function]}>
              Experiments
            </a>
          </div>
          <div
            className="column list">
            <a
              href="#">
              cases
            </a>
            <a
              href="#">
              methods
            </a>
            <a
              href="#">
              organizations
            </a>
            <a
              href="#">
              Users
            </a>
          </div>
          <div
            className="column list">
            <a
              href="#">
              News
            </a>
            <a
              href="?help"
              onClick={[Function]}>
              Help
            </a>
            <a
              href="#">
              Contact
            </a>
          </div>
        </div>
        <div
          className="col col-lg-6">
          <div
            className="column xl">
            <h4>
              Participedia
            </h4>
            <p>
              A global community sharing knowledge and stories about public participation
            </p>
          </div>
          <div
            className="column right">
            <div
              className="SocialMediaShareButton SocialMediaShareButton--facebook social-icons"
              onClick={[Function]}
              style={Object {}}>
              <img
                alt=""
                src="pp-social-fb.png" />
            </div>
            <div
              className="SocialMediaShareButton SocialMediaShareButton--twitter social-icons"
              onClick={[Function]}
              style={Object {}}>
              <img
                alt=""
                src="pp-social-tw.png" />
            </div>
            <div
              className="copyright-area">
              <img
                alt=""
                src="cc-icon.png" />
              <span>
                Participedia 2017
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
